src/web/functions.php

```diff
@@ -11,7 +11,232 @@
  */
 function getUniqueFirstLetters(array $airports)
 {
-    // put your logic here
+    $letters = [];
+    foreach ($airports as $var) {
+        $letters[] = getFirstLetter($var);
+    }
+    $letters = array_unique($letters);
+    sort($letters);
 
-    return ['A', 'B', 'C'];
+    return $letters;
+}
+
+/**
+ * Returns first letter of airports array item name
+ *
+ * @param $var
+ * @return mixed
+ */
+function getFirstLetter($var)
+{
+
+    return $var['name'][0];
+}
+
+/**
+ * Return airport array filtered by first letter
+ *
+ * @param $array
+ * @param $key
+ * @return array
+ */
+function filterByFirstLetter($array, $key)
+{
+    define('FIRST_LETTER', $key);
+
+    return array_filter($array, 'filterFirstLetter');
+}
+
+/**
+ * Callback function for filtering by first letter
+ *
+ * Use constant FIRST_LETTER as key
+ *
+ * @param $var
+ * @return bool
+ */
+function filterFirstLetter($var)
+{
+
```

Comment:
not needed empty line before return if it is single line in block

Reply:
Updated